R/zchunk_L233.water.demand.livestock.R

```diff
@@ -1,20 +1,21 @@
 #' module_water_L233.water.demand.livestock
 #'
-#' Briefly describe what this chunk does.
+#' This chunk is to generate water coefficients for region-specific livestock for model years.
 #'
 #' @param command API command to execute
 #' @param ... other optional parameters, depending on command
 #' @return Depends on \code{command}: either a vector of required inputs,
 #' a vector of output names, or (if \code{command} is "MAKE") all
 #' the generated outputs: \code{L233.TechCoef}. The corresponding file in the
 #' original data system was \code{L233.water.demand.livestock.R} (water level2).
-#' @details Describe in detail what this chunk does.
+#' @details This chunk is to generate water coefficients for region-specific livestock for
```

Comment:
"This chunk generates..."

Please define "water coefficients" more precisely. What does this mean?